### PR TITLE
implement built-in weighting calculations

### PR DESCRIPTION
**Pull request by kendalynnm**
Fun exercise which generated two primary questions:

1) Testing the package fails do to Nm and Nd not being used, despite these being removed as input parameters
2) Was not able to test this on the grossMethane code due to pdr_fractionation.rda issue, gar! Not sure if this is a OneDrive issue or an input type issue?

### CODE REVIEW

File: R/cost_function.R [hunk 1]
```diff
@@ -38,8 +36,11 @@ cost_function <- function(params, time, m, n, Nm, Nd,
                         frac_P = frac_P,
                         frac_k = frac_k)
 
+  pool_weight = sd(m) / m_prec # Normalization factor for pool size, see Eq. 12
+  ap_weight = sd((n / n + m) * 100) / ap_prec # Normalization factor for isotopic signature, see Eq. 13
```
Comment (bpbond) [inline R/cost_function.R:40]: @kendalynnm Hmm this looks suspicious; `n / n + m` is just `1 + m` ...are the parentheses misplaced?

**Review by bpbond — CHANGES_REQUESTED** [review]
Looks great! See below for one question about an equation.

FYI, there were two issues causing errors:
* You had changed the `cost_function` parameter names, and changed their roxygen entries above the function, but hadn't rebuild the package documentation
* The `test-cost_function.R` file had some usages of `Nm` and `Nd` that needed to be updated

I have pushed c212b3c to fix these.

File: R/cost_function.R (hunk 1 above)
Comment (kendalynnm) [reply R/cost_function.R:40]: Good catch, yes

Comment (bpbond) [reply R/cost_function.R:40]: ☝️ Once this is fixed, I would say go ahead and merge!

**Comment by bpbond** [comment]
Hmm it looks like you merged this without fixing the equation?

**Comment by kendalynnm** [comment]
> Hmm it looks like you merged this without fixing the equation?

So...truth is I don't remember how to change the code within the review interface (I also didn't put a lot of effort into figuring it out). My intent is to change it with my next commit to the master. I'm very happy to revise this strategy now and in the future.

**Pull request merged**